### PR TITLE
[86bzrjwbq][website] added navbar theme toggler and social links hints (tooltips)

### PR DESCRIPTION
**Pull request by msereniti**
## Motivation and Context

This is just needed to meet formal a11y requirments.

## How has this been tested?

Manually.

## Screenshots:
Screenshot 2024-08-06 at 12 39 56

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [x] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Review by sheila-semrush — CHANGES_REQUESTED** [review]
Looks good. I'd try adding a 200ms delay before showing the Hint.

File: website/docs/.vitepress/theme/Tooltip.vue [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+.VPNavScreen .vp-tooltip {
+  display: none;
+}
+.VPMenu:has(.vp-tooltip) {
+  overflow: visible;
+}
+.appearance-action:has(.vp-tooltip) {
+  margin-right: -2px;
+  display: flex;
+  align-items: center;
```
Comment (sheila-semrush) [inline website/docs/.vitepress/theme/Tooltip.vue:80]:
```suggestion
  align-items: center;
}
```

**Review by sheila-semrush — CHANGES_REQUESTED** [review]
Found several issues:

- [ ] Tooltip stays visible forever after using the switcher:

image

- [ ] In the intermediate adaptive state, the dropdown content is cropped and has scrollbars:

image

- [ ] In the intermediate adaptive state, controls still have their native titles, I guess we should remove them?

image

- [ ] We lost the new tooltips in the smallest adaptive state, but the native titles are still there:

image

- [ ] Let's add back the sun/moon icon inside the switch:

image

File: website/docs/.vitepress/theme/Tooltip.vue (hunk 1 above)
Comment (msereniti) [reply website/docs/.vitepress/theme/Tooltip.vue:82]: >  We lost the new tooltips in the smallest adaptive state

There is no point in providing tooltips for mobile devices

**Review by sheila-semrush — APPROVED** [review]

**Pull request merged**